modules/wri_publication/templates/wri-publication-columns.html.twig

```diff
@@ -99,6 +99,7 @@
 
             {% if content.share %}
             <div {{ region_attributes.share.addClass('layout__region', 'layout__region--share') }}>
+                <div class="mobile-social visit-on-social margin-right-md">{{ content.sidebar|without('field_narrative_taxonomy') }}</div>
```

Comment:
In this case I think you've got to just use the individual field. Check out https://pr-1314-wriflagship.pantheonsite.io/research/achieving-replacement-level-fertility

<img width="1435" alt="Screenshot 2025-06-25 at 8 08 49 AM" src="https://github.com/user-attachments/assets/50919855-8bd5-4ca7-9ef5-ecb120c736a0" />